sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/InvocationVisitor.cs

```diff
@@ -107,14 +107,21 @@ private ProgramState HandleStringNullOrWhiteSpaceCheckMethod()
         {
             var newProgramState = this.programState
                 .PopValue(out var arg1)
-                .PopValue();
+                .PopValue(out var arg2);

```

Comment:
Q: why does it have 2 arguments?
A: one is the relationship, one is the arg for comparison

can you please give better names for `arg1` and `arg2`? arg1 is argumnet; arg2 is null or whitespace